python/py-bcrypt/Portfile

```diff
@@ -6,11 +6,11 @@ PortGroup           select 1.0
 
 name                py-bcrypt
 version             4.0.1
-revision            0
+revision            1
```

Comment:
no need to increase the "revision" as there are no changes to existing subports. The new addition of py311 will be build regardless.